_posts/2024-03-29-geospatial-techno.md

```diff
@@ -0,0 +1,78 @@
+- Format: Default options are **SLD** and **ZIP** formats. To use other formats such as **CSS** and **YSLD**, you should download and install extensions. Make sure to match the version of the extension to the version of the GeoServer.
+
+The "Style Content" area provides options for creating, copying, or uploading a style. It has three options:
+- Generate a default style: Choose a generic style based on geometry such as Point, Line, Polygon, Raster, or Generic and click the **Generate** link when selected.
+- Copy from existing style: Select an existing style from GeoServer and copy its contents to the current style. Note that not all styles may be compatible with all layers. Click the **Copy** link when selected.
+- Upload a style file: Press the **Browse** button to locate and select a plain text file from your local system to add as the style. Click on the **Upload** link to add the style file.
+
+The **Legend** area allows you to preview the legend for the style. Click on the **Preview legend** link to generate a legend based on the current settings.
+
+At the bottom of the Style Editor page, you'll find several options: **Validate**, **Apply**, **Save** and **Cancel**. During editing and especially after editing is complete, you can check the validation of the syntax by pressing the **Validate** button at the bottom. If any validation errors are found, they will be displayed a red message and if no errors are found, you will see the green message. To make changes, press the **Apply** button to access all the tabs and finally press the **Save** button.
+
+After having created the style, it's time to apply it to the layer. To do it, follow these steps:
+- Navigate to the **Data > Layers** page then click on the **layer's name** link to open the layer's properties form. Switch to the Publishing tab.
+- Go to the **Style** section and from the **Default Style list**, select the suitable style, then press the **Save** button.
+- Navigate to the **Data > Layer Preview ** page and open up OpenLayers preview for the layer.
+
+## Edit a Style
+On the Styles page, click on the style name to open the **Style Editor**. The Style Editor page presents the style definition and contains four tabs with many configuration options: **Data** , **Publishing** , **Layer Preview** and **Layer Attributes**.
+- Data tab: The Data tab includes basic style information, the ability to generate a style, and legend details. Moreover, it allows for direct editing of style definitions at the bottom, with support for line numbering, automatic indentation, and real-time syntax highlighting. Users can switch between tabs to create and edit styles easily and can adjust the font size of the editor.
```

Comment:
The style of the wording changes here. In previous sections, it was in second person (e.g. "you"). Now it is an anonymous third person (e.g. "Users can"). A consistent approach would read better.

Reply:
It's done